openwisp_monitoring/check/tests/test_iperf.py

```diff
@@ -469,3 +527,143 @@ def test_iperf_check(self):
                 mock_get_iperf_servers.assert_called_once_with(
                     self.device.organization.id
                 )
+
+    def test_iperf_check_auth_config(self):
```

Comment:
I would declare the common mocks with decorators to reduce code duplicity. 
```suggestion
    @patch.object(Iperf, '_get_iperf_servers', return_value=['iperf.openwisptestserver.com'])
    @patch.object(Iperf, '_exec_command', side_effect=[(RESULT_TCP, 0), (RESULT_UDP, 0)])
    @patch.object(iperf_logger, 'warning')
    def test_iperf_check_auth_config(self, mock_warn, mock_exec_command, mock_get_iperf_servers):
```